README.rst

```diff
@@ -9,70 +9,89 @@ Open Zaak
 
-**Open Zaak** is een moderne, open source gegevens- en services-laag om
-`zaakgericht werken`_ te ondersteunen zoals voorgesteld in de onderste 2 lagen
-van het `Common Ground`_ 5-lagen model.
+Daarnaast zijn er een paar generieke API's die nodig om gebruik te maken van deze API's:
 
-De gegevens worden ontsloten middels een `gestandaardiseerde set VNG API's`_,
-te weten:
+* Notificaties - in Common Ground worden gegevens bij de bron geregistreerd en bijgehouden. Consumers krijgen niet vanzelf bericht als er iets is gewijzigd. Hiervoor kunnen ze een abonnement registreren bij de Notificaties API.
+* Autorisaties - via de Autorisaties API wordt de toegang van applicaties tot gegevens geregeld.
 
-* Zaken API (`Zaken API-specificatie 1.0`_)
-* Documenten API (`Documenten API-specificatie 1.0`_)
-* Catalogi API (`Catalogi API-specificatie 1.0`_)
-* Besluiten API (`Besluiten API-specificatie 1.0`_)
-* Autorisaties API (`Autorisaties API-specificatie 1.0`_)
+Productiewaardige API's
+==========
 
-De `Notificaties API`_ is nodig voor **Open Zaak** en is beschikbaar via
-`Open Notificaties`_.
+Bij de realisatie van productiewaardige API's is aandacht besteed aan een aantal belangrijke aspecten:
 
-.. _`Common Ground`: https://commonground.nl/
-.. _`zaakgericht werken`: https://www.vngrealisatie.nl/ondersteuningsmiddelen/zaakgericht-werken
-.. _`gestandaardiseerde set VNG API's`: https://zaakgerichtwerken.vng.cloud/
-.. _`Zaken API-specificatie 1.0`: https://zaakgerichtwerken.vng.cloud/standaard/zaken/index
-.. _`Documenten API-specificatie 1.0`: https://zaakgerichtwerken.vng.cloud/standaard/documenten/index
-.. _`Catalogi API-specificatie 1.0`: https://zaakgerichtwerken.vng.cloud/standaard/catalogi/index
-.. _`Besluiten API-specificatie 1.0`: https://zaakgerichtwerken.vng.cloud/standaard/besluiten/index
-.. _`Autorisaties API-specificatie 1.0`: https://zaakgerichtwerken.vng.cloud/standaard/autorisaties/index
-.. _`Notificaties API`: https://zaakgerichtwerken.vng.cloud/standaard/notificaties/index
-.. _`Open Notificaties`: https://github.com/open-zaak/open-notificaties
+* Beheer: er is een beheerportaal ingericht waarmee de verschillende API's door functioneel beheerders kunnen worden beheerd.
+* Performance: er zijn performance-metingen verricht op basis van schattingen van de verwachte belasting door applicaties die eindgebruikers gebruiken. Benodigde verbeteringen zijn doorgevoerd waardoor een belasting door 2000 eindgebruikers geen problemen zou moeten opleveren.
+* Documentatie van de componenten, met name van de beheer applicaties. (De inhoudelijke documentatie over de API's is onderdeel van de standaard.)
+* Uitrol: Om de uitrol naar servers te vereenvoudigen zijn er Docker containers en een Virtual Machine (VM) image beschikbaar. Dit zijn een soort componenten die gemakkelijk kunnen worden uitgerold op een server om ze vervolgens in gebruik te nemen. Hiermee kunnen gemeenten de API’s op eenvoudige wijze (laten) draaien bij een hostingpartij.
```

Comment:
Niet helemaal correct. Er is geen VM image beschikbaar. Er is een Docker container, documentatie en installie scripts om het uit te rollen op Kubernetes, een server of VMware.